plugins/mac-avcapture/av-capture.mm

```diff
@@ -2236,7 +2237,28 @@ static void add_manual_properties(obs_properties_t *props)
 		OBS_COMBO_FORMAT_STRING);
 	obs_property_list_add_string(dev_list, "", "");
 
-	for (AVCaptureDevice *dev in [AVCaptureDevice devices]) {
+	NSArray *devices = nil;
+#if (__MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_15)
+	if (@available(macOS 10.15, *)) {
+		AVCaptureDeviceDiscoverySession *deviceDiscoverySession = [AVCaptureDeviceDiscoverySession
+			discoverySessionWithDeviceTypes:@[
+				AVCaptureDeviceTypeBuiltInWideAngleCamera,
+				AVCaptureDeviceTypeExternalUnknown
+			]
+					      mediaType:AVMediaTypeVideo
```

Comment:
This will need `AVMediaTypeMuxed` as well so that we still pick up the screen of connected i(Pad)OS devices (and other cameras that could potentially have sound)

Reply:
That would require a second discovery session, merging the results of both arrays, as `AVCaptureDeviceDiscoverySession` only allows a single `mediaType` value.

Reply:
FWIW, implemented just that - got my iPhone camera as well as the built-in Studio Display camera.

Will push this once we cleared up the other comments.

Reply:
@gxalpha pushed an update that should now include muxed devices in the list again.